src/pal/src/exception/seh-unwind.cpp

```diff
@@ -491,7 +491,17 @@ BOOL PAL_VirtualUnwindOutOfProc(CONTEXT *context,
 
     LibunwindCallbacksInfo.Context = context;
     LibunwindCallbacksInfo.readMemCallback = readMemCallback;
+#if UNWIND_CONTEXT_IS_UCONTEXT_T
     WinContextToUnwindContext(context, &unwContext);
+#else
+    st = unw_getcontext(&unwContext);
```

Comment:
You are right, in this case, the context pointers cannot ever point to the unwContext. But to not to burden the GetContextPointers with the check to NULL, we can pass in a dummy unwContext which we would never try to fill in (neither with the WinContextToUnwindContext nor with the unw_getcontext). So your fix can effectively be just removing of the WinContextToUnwindContext call.